contracts/Console.sol

```diff
@@ -0,0 +1,687 @@
+// SPDX-License-Identifier: MIT
```

Comment:
LGTM - can you please move this to a different folder? i.e. something like /utils/debug/console.log?